searx/shared/__init__.py

```diff
@@ -1,11 +1,14 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 import logging
+import importlib
 
 logger = logging.getLogger('searx.shared')
 
+__all__ = ['SharedDict', 'schedule']
+
 try:
-    import uwsgi
+    uwsgi = importlib.import_module('uwsgi')
 except:
     # no uwsgi
```

Comment:
@dalf Strange, how could this code had been worked before?  

BTW: this points out why catching general Exception is worse  / better we catch only `ModuleNotFoundError` .. 

Issues like this are reported by pylint / we should add a

    # lint: pylint

comment on top of the file.